### PR TITLE
Typo correction re Bug 1032143

### PR DESCRIPTION
**Pull request by cjschroder**
@taroth21 @fsundermeyer @dmpop Would you look at the bug report? I'm pretty sure the only issue with this section is the extra -d (crowbar-d), which I corrected in this PR. The rest of the bug report doesn't seem to apply.

https://bugzilla.suse.com/show_bug.cgi?id=1032143

### CODE REVIEW

**Comment by taroth21** [comment]
@cjschroder : Thanks for the PR! However, I think the following information from the bug report does apply - please see my comments to the file.

File: xml/depl_conf_admin_crowbar.xml [hunk 1]
```diff
@@ -1442,13 +1442,13 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
      Apart from the roles listed under <xref linkend="ex.conduits.role"/>
      each node in &cloud; has a unique role, which lets you create modes
      matching exactly one node. The role is named after the scheme
-     <literal>crowbar-d <replaceable>FULLY QUALIFIED
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
```
Comment (taroth21) [inline xml/depl_conf_admin_crowbar.xml]: If I understood the bug report correctly, this should be changed to: The FULLY QUALIFIED HOSTNAME for all nodes booted via PXE is composed of the following: a prefix 'd', the MAC address of the network interface used to boot the node via PXE, and the domain name configured on the &admserv;.

File: xml/depl_conf_admin_crowbar.xml [hunk 2]
```diff
@@ -1442,13 +1442,13 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
...
      network interface used to boot the node via PXE, and the domain name
      configured on the &admserv;. Colons and periods are replaced with
-     underscores. An example role name would be:
-     <literal>crowbar-d1a-12-05-1e-35-49_my_cloud</literal>.
+     underscores. So the role name derived from the configuration example (below) is:
+     <literal>crowbar_d1a-12-05-1e-35-49_my_cloud</literal>.
```
Comment (cjschroder) [inline xml/depl_conf_admin_crowbar.xml]: Yes, my mistake.

File: xml/depl_conf_admin_crowbar.xml [hunk 3]
```diff
@@ -1442,13 +1442,13 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
...
      matching exactly one node. The role is named after the scheme
-     <literal>crowbar-d <replaceable>FULLY QUALIFIED
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
      network interface used to boot the node via PXE, and the domain name
      configured on the &admserv;. Colons and periods are replaced with
-     underscores. An example role name would be:
-     <literal>crowbar-d1a-12-05-1e-35-49_my_cloud</literal>.
+     underscores. So the role name derived from the configuration example (below) is:
```
Comment (cjschroder) [inline xml/depl_conf_admin_crowbar.xml]: Ah, I had it backwards-- I'll revise it.

**Comment by cjschroder** [comment]
I've been working on github-hosted projects for a couple years and I still get befuddled by the commenting system. At any rate, @taroth21 I think I got it this time. Also note "Colons and periods are replaced with underscores". Thanks very much for your help.

File: xml/depl_conf_admin_crowbar.xml [hunk 4]
```diff
@@ -1439,17 +1439,16 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
...
-     <literal>crowbar-d <replaceable>FULLY QUALIFIED
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
-     HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
-     network interface used to boot the node via PXE, and the domain name
-     configured on the &admserv;. Colons and periods are replaced with
-     underscores. An example role name would be:
-     <literal>crowbar-d1a-12-05-1e-35-49_my_cloud</literal>.
+     HOSTNAME</replaceable> in turn is composed of the following: a prefix 'd', the MAC address of the network interface used to boot the node via PXE, and the domain name configured on the &admserv;. Colons and periods are replaced with
+     underscores. An example role name would be:
```
Comment (fsundermeyer) [inline xml/depl_conf_admin_crowbar.xml]: This needs to be changed - I already had it wrong in the original text. It mixes how the FQDN and the role name are created, and contains misleading information.

* the FQDN is created by prefixing "d" to the MAC address and replacing colons with dashes
* the role name is created by prefixing the FQDN with "crowbar-" and replacing the periods in the domain name with underscores (the host name part is _not_ changed)

File: xml/depl_conf_admin_crowbar.xml [hunk 5]
```diff
@@ -1439,17 +1439,16 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
...
+     <literal>crowbar-<replaceable>FULLY QUALIFIED
      HOSTNAME</replaceable></literal>. The <replaceable>FULLY QUALIFIED
-     HOSTNAME</replaceable> in turn is composed of the following: the MAC address of the
-     network interface used to boot the node via PXE, and the domain name
-     configured on the &admserv;. Colons and periods are replaced with
-     underscores. An example role name would be:
-     <literal>crowbar-d1a-12-05-1e-35-49_my_cloud</literal>.
+     HOSTNAME</replaceable> in turn is composed of the following: a prefix 'd', the MAC address of the network interface used to boot the node via PXE, and the domain name configured on the &admserv;. Colons and periods are replaced with
+     underscores. An example role name would be: 
+     <literal>crowbar-d1a_12_05_1e_35_49_my_cloud</literal>.
```
Comment (fsundermeyer) [inline xml/depl_conf_admin_crowbar.xml]: crowbar-d1a-12-05-1e-35-49_my_cloud

File: xml/depl_conf_admin_crowbar.xml [hunk 6]
```diff
@@ -1469,7 +1468,7 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
          ],
          "conduit_map" : [
           {
-            "pattern" : "my_mode/.*/crowbar-d1a-12-05-1e-35-49_my_cloud",
+            "pattern" : "my_mode/.*/crowbar-d1a_12_05_1e_35_49_my_cloud",
```
Comment (fsundermeyer) [inline xml/depl_conf_admin_crowbar.xml]: crowbar-d1a-12-05-1e-35-49_my_cloud

File: xml/depl_conf_admin_crowbar.xml [hunk 7]
```diff
@@ -1483,6 +1482,11 @@ lrwxrwxrwx 1 0 Jun 19 08:43 eth3 -&gt; ../../devices/pci0000:00/0000:00:1c.0/000
    }
 }</screen>
 
+    <para>
+      The <literal>crowbar-<replaceable>FULLY QUALIFIED
+              HOSTNAME</replaceable></literal> scheme also applies to admin nodes, which are not booted from PXE. However, the details of the scheme are different than nodes which boot from PXE: there is no prefix d, and the &admserv; should have a <replaceable>admin.cloud.&exampledomain;</replaceable>-type FQDN, which is used in place of the MAC address. So the pattern in your conduit map looks like <literal>my_mode/.*/crowbar-admin_cloud_example_com_my_cloud</literal>
+</para>
+
```
Comment (fsundermeyer) [inline xml/depl_conf_admin_crowbar.xml]: For the (there is only one) admin server and all nodes registered manually, a different naming scheme for the role name applies:

crowbar-&lt;replaceable>FQDN&lt;/replaceable>

where periods in  &lt;replaceable>FQDN&lt;/replaceable> are replaced by underscores

Consider moving this information up

Comment (cjschroder) [inline xml/depl_conf_admin_crowbar.xml]: Thanks @fsundermeyer, and for asking for clarification on the bug report. I'll correct this, and look for other locations in the manual where it should go.

**Comment by cjschroder** [comment]
Hopefully this is the final version!

**Pull request merged**